sph/include/sph/eos.hpp

```diff
@@ -9,41 +9,80 @@ namespace sphexa
 namespace sph
 {
 
+/*! @brief Reduced version of Ideal gas EOS for internal energy
+ *
+ * @param u    internal energy
+ * @param rho  baryonic density
+ *
+ * This EOS is used for simple cases where we don't need the temperature.
+ * Returns pressure, speed of sound
+ */
 template<class T1, class T2>
-CUDA_DEVICE_HOST_FUN util::tuple<T2, T2> equationOfState(T1 u, T2 rho)
+CUDA_DEVICE_HOST_FUN auto equationOfState(T1 u, T2 rho)
```

Comment:
maybe `idealGasEOS` and `polytropicEOS` would be cleaner names?

Reply:
Sure! I will do this change.